pkg/rancher/migrations.go

```diff
@@ -123,6 +132,33 @@ func createOrUpdateConfigMap(configMapClient controllerv1.ConfigMapClient, cm *v
 	return err
 }
 
+func versionTombstone(configMapController controllerv1.ConfigMapController) error {
+	cm, err := getConfigMap(configMapController, rancherVersionTombstoneConfig)
+	if err != nil || cm == nil {
+		return err
+	}
+
+	if !semver.IsValid(rancherversion.Version) {
+		logrus.Errorf("rancher version %s is not semver compliant, skipping tombstone validation", rancherversion.Version)
```

Comment:
Is a warning-level message more suitable in this case?

Reply:
I kind of want them to both stay as an error. If someone starts messing with their Rancher version so that it's not semver compatible, they are liable to break more than just this migration. They can break the UI, and charts versioning will fail resulting in all values being returned, the channelserver will return incorrect values, steve is liable to return the incorrect version, etc. My hope is that an error like this will show up in whatever log collector is being used.

Reply:
OK. It makes sense to me.